src/hosting/api.ts

```diff
@@ -460,13 +460,14 @@ export async function cloneVersion(
 export async function createRelease(
   site: string,
   channel: string,
-  version: string
+  version: string,
+  partialRelease?: Partial<Pick<Release, "message" | "type">>
```

Comment:
q: I notice that types don't necessarily match here and in the `.post` call:

```
Partial<Pick<Release, "message" | "type">>
Partial<Pick<Release, "message" >>
```

Intentional? Do you want to define an intermediate type instead of retyping this fairly long type def?

Reply:
Yeah, meant to make a new type, then didn't. Thanks.